### DISCUSSION TITLE
Welcome to zinit Discussions!

### DISCUSSION THREAD

**Discussion by alichtman** [zdharma-continuum/zinit#13] [Announcements]
* Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Conduct yourself respectfully + reasonably.

**Comment by neumachen** [comment]
I am curious about what happened. I only found out because I was booting up a new machine and it wouldn't pull the repo anymore. Weird.

**Reply by pschmitt** [reply]
No one really knows what went through his mind when psp decided to delete the zdharma org.
We may very well never know, I don't believe anyone from the community was able to get in touch with him.

**Reply by alichtman** [reply]
I sent him an email and it bounced. It looks like his domain was shut down.
